### ISSUE TITLE
Timezone information of date fields are not properly parsed

### ISSUE THREAD

**Issue by coezbek** [greatseth/mediainfo#52]
It seems that timezone information of date fields are not properly parsed, because Time.parse does not understand the prefix "UTC" in the date strings. It just ignores it and returns a local time.

Ways to reproduce

```
require 'mediainfo'
require 'active_support/core_ext/hash'
    
source = .... # Your file here
mi = MediaInfo.from(source)
    
puts "Returned from MediaInfo: #{mi.general.encoded_date}"
    
xmldate = Hash.from_xml(mi.xml)["MediaInfo"]["media"]["track"][0]["Encoded_Date"]
    
puts "Raw date from MediaInfo xml: #{xmldate}"
puts "Time parse directly on xmldate: #{Time.parse(xmldate)}"
puts "Time parse with UTC correction: #{Time.parse(xmldate.gsub("UTC\s+", '') + " UTC").localtime}"
```

This will return (for one example):

```
Returned from MediaInfo: 2017-08-09 23:08:00 +0200 -> Wrong
Raw date from MediaInfo xml: UTC 2017-08-09 23:08:00 -> This is what is in the file.
Time parse directly on xmldate: 2017-08-09 23:08:00 +0200 -> Wrong
Time parse with UTC correction: 2017-08-10 01:08:00 +0200 -> This should be returned
```

I just noticed this, because I was checking filenames (which included dates) against metadata using MediaInfo gem and because I am in UTC+2 there was this file where there was a different date returned by MediaInfo compared to the filename.

**Comment by NorseGaud** [comment]
@coezbek  Got an example file you can upload for me to use for testing?

**Comment by rlue** [comment]
@NorseGaud

https://private.ryanlue.com/R0022792.MOV

```sh
$ mediainfo ~/tmp/R0022792.MOV | grep 2021
Encoded date                             : UTC 2021-05-03 15:48:43
Tagged date                              : UTC 2021-05-03 15:48:43
Encoded date                             : UTC 2021-05-03 15:48:43
Tagged date                              : UTC 2021-05-03 15:48:43
Encoded date                             : UTC 2021-05-03 15:48:43
Tagged date                              : UTC 2021-05-03 15:48:43

$ ruby -r'mediainfo' -e 'puts MediaInfo.from("/home/rlue/tmp/R0022792.MOV").video.encoded_date'
2021-05-03 15:48:43 -0700
```

**Comment by rlue** [comment]
FYI this problem does not appear to happen with DNGs (note that the `-0700` time zone is not applied in this case):

```sh
$ curl -O https://private.ryanlue.com/2021-03-26_223527.dng
$ mediainfo 2021-03-26_223527.dng | grep date
Encoded date                             : 2021:03:26 22:35:27
$ ruby -r'mediainfo' -e 'puts MediaInfo.from("2021-03-26_223527.dng").image.encoded_date'
2021:03:26 22:35:27
```

**Comment by NorseGaud** [comment]
Released in 1.5.0
